x/stakeibc/keeper/lsm_token_deposit.go

```diff
@@ -0,0 +1,51 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
+)
+
+func (k Keeper) SetLSMTokenDeposit(ctx sdk.Context, deposit types.LSMTokenDeposit) {
+	// TODO [LSM]
+}
+
+func (k Keeper) RemoveLSMTokenDeposit(ctx sdk.Context, denom string) {
+	// TODO [LSM]
+}
+
+func (k Keeper) GetLSMTokenDeposit(ctx sdk.Context, denom string) (deposit types.LSMTokenDeposit, found bool) {
+	// TODO [LSM]
+	return
+}
+
+func (k Keeper) GetAllLSMTokenDeposit(ctx sdk.Context) []types.LSMTokenDeposit {
+	// TODO [LSM]
+	return []types.LSMTokenDeposit{}
+}
+
+func (k Keeper) AddLSMTokenDeposit(ctx sdk.Context, deposit types.LSMDepositStatus) {
+	// TODO [LSM]
+	// See if a deposit already exists for this denom
+	// If so, increment the amount
+	// otherwise, create a new deposit
+}
+
+func (k Keeper) UpdateLSMTokenDepositStatus(ctx sdk.Context, deposit types.LSMDepositStatus, status types.LSMDepositStatus) {
```

Comment:
Wrong type on variable deposit for this and 2-3 other methods here.  IDE probably autocompleted LSMDepositStatus but any of the variables named deposit are supposed to be LSMTokenDeposit types.

Reply:
done